src/components/Datagrid/useDatagridChange.ts

```diff
@@ -1,5 +1,6 @@
 import { updateAtIndex } from "@dashboard/utils/lists";
 import { EditableGridCell, Item } from "@glideapps/glide-data-grid";
+import { map } from "lodash-es";
```

Comment:
**nitpick:** vite probably does tree-shaking anyway so this wouldn't make a difference, but just to be sure I'd import lodash packages like this
```suggestion
import map from "lodash/map";
```